tenable/io/v3/vm/folders/api.py

```diff
@@ -79,20 +83,95 @@ def edit(self, id: int, name: str) -> None:
+                Examples:
+                    >>> ('or', ('and', ('test', 'oper', '1'),
+                    ...            ('test', 'oper', '2')
+                    ...     ),
+                    ... 'and', ('test', 'oper', 3)
+                    ... )
+                    >>> {'or': [
+                    ...         {'and': [
+                    ...                 {
+                    ...                     'value': '1',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '1'
+                    ...                 },
+                    ...                 {
+                    ...                     'value': '2',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '2'
+                    ...                 }
+                    ...             ]
+                    ...         }
+                    ...     ],
+                    ... 'and': [
+                    ...         {
+                    ...             'value': '3',
+                    ...             'operator': 'oper',
+                    ...             'property': 3
+                    ...         }
+                    ...     ]
+                    ... }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth:`tio.v3.vm.definitions.folders()`
+                endpoint to get more details.
+            sort list(tuple, Dict):
+                A list of dictionaries describing how to sort the data
+                that is to be returned.
+                Examples:
+                    - [('field_name_1', 'asc'),
+                             ('field_name_2', 'desc')]
+                    - [{'property': 'last_observed', 'order': 'desc'}]
+            limit (int):
+                Number of objects to be returned in each request.
+                Default is 200.
+            next (str):
+                The pagination token to use when requesting the next page of
+                results.  This token is presented in the previous response.
+            return_resp (bool):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool):
+                If set to true, It wil return the CSV Iterable. Returns all
+                data in text/csv format on each next call with row headers
+                on each page.
         Returns:
-            :obj:`list`: List of folder resource records.
-
+            Iterable:
+                The iterable that handles the pagination and potentially
+                async requests for the job.
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
```

Comment:
Update docstring to describe correct behavior.

Reply:
Updated